src/components/Popover/Popover.stories.tsx

```diff
@@ -11,24 +11,26 @@ export default {
     layout: 'centered',
     badges: [BADGE.BETA],
   },
-} as Meta;
+};
 
-const Template: Story<Props> = (args) => <PopoverExample {...args} />;
+type Args = React.ComponentProps<typeof PopoverExample>;
 
-export const Default = Template.bind({});
-Default.args = {};
+export const Default: StoryObj<Args> = {};
 
-export const TopLeft = Template.bind({});
-TopLeft.args = {
-  position: 'top-left',
+export const TopLeft: StoryObj<Args> = {
+  args: {
+    position: 'top-left',
+  },
 };
 
-export const BottomLeft = Template.bind({});
-BottomLeft.args = {
-  position: 'bottom-left',
+export const BottomLeft: StoryObj<Args> = {
+  args: {
+    position: 'bottom-left',
+  },
 };
 
-export const BottomRight = Template.bind({});
-BottomRight.args = {
-  position: 'bottom-right',
+export const BottomRight: StoryObj<Args> = {
+  args: {
+    position: 'bottom-right',
+  },
 };
```

Comment:
I think it should, yeah. I think that's true for all popover-type components. I tested this component too and it looks like right now there's actually no way to close it?

Reply:
Can you get to the trigger button using a keyboard? I can't figure out how to get out of the popover.

Reply:
UHHH I definitely cannot get to the trigger button using a keyboard. Should that be a separate ticket or part of the escape to close ticket's description?

Reply:
I'm mostly saying that right now it looks like you can't close the popover at all using a keyboard because you can't get back to the trigger. So this issue should be launch-blocking for this component as it is not accessible for keyboard users. Once we add the escape key logic, users will be able to get back to the trigger using that mechanism.

Reply:
Made a PR to address this need #1137

Reply:
Awesome thanks!